toolkits/global/packages/global-corporate-footer/README.md

```diff
@@ -1,29 +1,155 @@
+
+The privacy policy you need to use to depends on the legal entity for the product you're working on. Talk to your Product Owner or Manager to find out which privacy policy to link to.
+
+### Extended footer
+
+The extended footer contains the same links as the default footer, with the option to include:
+
+- California privacy statement
+- Terms and conditions
+- Help and support
+
+#### California privacy statement and Help and support links
+
+Some products need to include one or both of the following:
+
+- California privacy statement
+- Help and support
+
+If you need to include a California Privacy Statement link, you must use the url in the [component demo's `context.json` file](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/demo/context.json).
+
+Help and support should link to the relevant contact options your service offers.
+
+#### Terms and conditions
+
+The text you should use for this link depends on what it covers for your product or service. 
+
+The most common options are:
+
+- Terms and conditions
+- Terms of use
+- Impressum
+- Imprint
+
+## Installation
+
+To use the Corporate footer component, enter the following command in
+your Terminal:
+
+```
+npm install @springernature/global-corporate-footer
+```
+The Corporate footer component is designed to be the same across all brands. Because of this, it doesn't include any brand settings like some of the other components.
+
+Import the installed component code in your `scss` file:
 
 ```scss
-// Pick ONE of the brands below to include
 @import '@springernature/global-corporate-footer/scss/10-settings/default';
-@import '@springernature/global-corporate-footer/scss/10-settings/springer';
-@import '@springernature/global-corporate-footer/scss/10-settings/nature';
 
 // Include this with your other components
 @import '@springernature/global-corporate-footer/scss/50-components/corporate-footer';
 ```
 
-## Examples
+## Template
 
-### HTML
-```html
-<div class="c-corporate-footer">
-    <div class="u-container">
-        <img src="images/logos/sn-logo-white.svg" alt="Springer Nature" loading="lazy" width="140" height="14"/>
-        <p class="c-corporate-footer__legal" data-test="copyright">&copy; 2020 Springer Nature Limited</p>
-    </div>
-</div>
+Find a configurable template in the [`view` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/view). 
+
+You can see an example in the [`demo` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/demo).
+
+### Template configuration
+
+#### Using the Corporate footer with an existing footer
+
+The Corporate footer component doesn't include a `<footer>` region/landmark. This means you can use it together with - for example - `springer-nature-publisher` inside a common `<footer>`.
```

Comment:
@Heydon I've put this line back in and reworded it slightly - can you see if it looks OK?

Reply:
@amyhupe LGTM!